flink-kubernetes-operator-autoscaler/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/metrics/ScalingMetrics.java

```diff
@@ -82,7 +82,7 @@ public static void computeDataRateMetrics(
             scalingMetrics.put(ScalingMetric.TRUE_PROCESSING_RATE, trueProcessingRate);
             scalingMetrics.put(ScalingMetric.CURRENT_PROCESSING_RATE, numRecordsInPerSecond);
         } else {
-            LOG.error("Cannot compute true processing rate without numRecordsInPerSecond");
+            LOG.warn("Cannot compute true processing rate without numRecordsInPerSecond");
```

Comment:
Same as above, the logging is probably not the best way to bring this to the intended audience and warn should probably be mostly reserved for operator internal problems.

Reply:
We could trigger an event, but putting this into the status may be an overkill because this is not really actionable.

Reply:
Not actionable is also why I'm not sure about logging at warn level. Who is going to read these logs? Will logs be flooded with repeating entries that are unlikely to be addressed? I thought a message in the status (or one time event) might be more useful, after all the application would need to be modified to remedy the condition.